docroot/modules/custom/mass_hierarchy/mass_hierarchy.install

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/**
+ * @file
+ * Installation functions for Mass Hierarchy.
+ */
+
+use Drupal\node\Entity\Node;
+
+/**
+ * Implementations of hook_update_N().
+ */
+
+/**
+ * Remove records on nested set table from non-current revisions.
+ */
+function mass_hierarchy_update_8105() {
```

Comment:
I think we should start from 8101 instead of 8105.

Reply:
Done, thanks!